server/server.go

```diff
@@ -857,6 +858,11 @@ func (s *Server) SetReplicationConfig(cfg config.ReplicationConfig) error {
 	return nil
 }
 
+func (s *Server) isOnlyDefaultRule() bool {
+	rules := s.GetRaftCluster().GetRuleManager().GetAllRules()
+	return len(rules) == 1 && rules[0].GroupID == "pd" && rules[0].ID == "default"
```

Comment:
What if we overwrite this default rule?

Reply:
So what's the definition of 'default rule' ? 

I thought it was only defined by GroupID and ID.